camviewer_ui_impl.py

```diff
@@ -651,81 +649,6 @@ def closeEvent(self, event):
             self.dumpConfig()
         QMainWindow.closeEvent(self, event)
 
-    #
-    # OK, what is going on here?
-    #
-    # When we want to do something that could cause a resize, we:
-    #     - Call startResize(), which sets the size contraint to fixed and resizing to True.
-    #     - Do the resize/hide/show/whatever.  The DisplayImage has its sizeHint() set to our view size.
-    #     - Call finishResize().  If we are totally done with any enclosed operation, we call adjustSize()
-    #       to force the window to recalculate/relayout.
-    #
-    # When the resize actually happens, we get a resize event, but this seems to be before most things
-    # have settled down.  So we do a singleShot(0) timer to get a callback when everything is done.
-    #
-    #
-    #
-
-    def startResize(self):
-        if self.rcnt == 0:
-            self.layout().setSizeConstraint(QLayout.SetFixedSize)
-            self.resizing = True
-        self.rcnt += 1
-
-    def finishResize(self):
-        self.rcnt -= 1
-        if self.rcnt == 0:
-            self.adjustSize()
-
-    def resizeEvent(self, ev):
-        QTimer.singleShot(0, self.completeResize)
-
-    def completeResize(self):
-        self.layout().setSizeConstraint(QLayout.SetDefaultConstraint)
-        self.setMaximumSize(QSize(16777215, 16777215))
```

Comment:
Note that here we're setting sizes and not SizeHints, another place where I imagine there's cross talk / competing instructions